vision/google/cloud/vision/image.py

```diff
@@ -54,7 +54,7 @@ def __init__(self, client, content=None, filename=None, source_uri=None):
                 content = file_obj.read()
 
         if content is not None:
-            content = _bytes_to_unicode(b64encode(_to_bytes(content)))
+            content = _to_bytes(content)
```

Comment:
So the PB version of base64 encodes the data for us.

So for the PB version I need the content unencoded and for the HTTP version I need the data encoded.